tests/Unit/inc/Engine/Admin/Settings/Settings/sanitizeCallback.php

```diff
@@ -101,6 +100,31 @@ public function testShouldSanitizeFontPreloadEntries( $input, $expected ) {
 		);
 	}
 
+	/**
+	 * @dataProvider addExcludeCSSProvider
+	 */
+	public function testShouldSanitizeExcludeCSS( $original, $sanitized ) {
+		$this->stubWpParseUrl();
+
+		Functions\when( 'content_url' )->justReturn( 'http://example.org/wp-content/' );
+		Functions\when( 'get_rocket_i18n_uri' )->justReturn( [ 'http://example.org/' ] );
+
+		Functions\when( 'sanitize_text_field' )->returnArg();
+		Functions\when( 'rocket_valid_key' )->justReturn( true );
```

Comment:
@crystinutzaa You'll want to also mock the new `rocket_validate_css()` function too as it's a dependency to this callback but external to it.